### PR TITLE
import code-lint workflow job

### PR DESCRIPTION
**Pull request by kwanhur**
* upgrade actions/checkout@v3
* import staticcheck action@v1.0.0
* enable go vet
* ignore many used tips by `//lint:file-ignore` and `//lint:ignore`
* fix partial `err` and used return value

Resolved issue #1017

### CODE REVIEW

**Comment by kwanhur** [comment]
cc @iyangsj @mileszhang2016

File: bfe_balance/bal_slb/bal_rr.go [hunk 1]
```diff
@@ -34,6 +34,8 @@
 //      9 -1 -1  a selected; 9-7+5 -1+1  -1+1  -> 7  0  0
 //      7  0  0  a selected; 7-7+5 0+1   0+1   -> 5  1  1
 
+//lint:file-ignore U1000 Ignore field is unused
```
Comment (iyangsj) [inline bfe_balance/bal_slb/bal_rr.go:37]: 这条规则建议统一忽略（或修复）
U1000 Ignore func is unused

**Review by iyangsj — CHANGES_REQUESTED** [review]
见批注

File: bfe_balance/bal_slb/bal_rr.go (hunk 1 above)
Comment (kwanhur) [reply bfe_balance/bal_slb/bal_rr.go]: 已经统一在 `staticcheck.conf` 添加忽略规则 `"-U1000"`

**Pull request merged**